source/simulator/assemblers/stokes.cc

```diff
@@ -196,6 +197,12 @@ namespace aspect
       Assert (this->get_parameters().use_equal_order_interpolation_for_stokes == false,
               ExcNotImplemented());
 
+      const bool assemble_A_approximation = this->get_parameters().use_full_A_block_preconditioner == false &&
+                                            this->get_parameters().mesh_deformation_enabled == false;
+
+      if (assemble_A_approximation == false)
```

Comment:
I would think this should this be an assert instead (and this assembler should never be called in the first place). Do you agree?